api4/plugin_test.go

```diff
@@ -547,6 +547,12 @@ func TestGetInstalledMarketplacePlugins(t *testing.T) {
 				HomepageURL: "https://example.com/mattermost/mattermost-plugin-nps",
 				IconData:    "https://example.com/icon.svg",
 				DownloadURL: "https://example.com/mattermost/mattermost-plugin-nps/releases/download/v1.0.3/com.mattermost.nps-1.0.3.tar.gz",
+				Labels: []model.MarketplaceLabel{
```

Comment:
I'm 0/5 if it's worth to update all tests in `api4/plugin_test.go` to include `Labels`.

Reply:
+1, as long as there is one test like this one that tests `Labels`, that should suffice.